File: search-algorithms/linear-search/linearSearch.js
function linearSearch(arr, elem) {
  for(let i = 0; i < arr.length; i++) {
    if (arr[i] == elem) {
      return `The element is in the index: ${i}`
    }
  }

  return "Not found."
}

console.log(linearSearch([1,2,3,4,5,6], 7))

File: search-algorithms/linear-search/linearSearch.py
def linearSearch(arr, elem):
  index = 0
  for item in arr:
    if item == elem:
      return "The element is in the index: {}".format(index)
    index += 1
  return "Not found."


print(linearSearch([1,2,3,4,5,6], 7))